opencog/atoms/core/PutLink.cc

```diff
@@ -123,11 +123,28 @@ void PutLink::static_typecheck_values(void)
 		return;
 
 	size_t sz = _varlist.varseq.size();
-	Type vtype = _values->get_type();
+
+	Handle valley = _values;
```

Comment:
Just curious (especially as English isn't my mother tong), why variable name "valley"?

Reply:
It sounds like "value" but is spelled differently, making it easy to search for. I use search a lot, so having unique strings is important.  Also, having maximally-different strings is desirable. When everything is vthis and vthat and var vars vars it gets hard to read.